README.md

```diff
@@ -22,15 +22,18 @@
 Windows is not an explicitly supported environment, although where possible Palisade has been made compatible.  
 For Windows developer environments, we recommend setting up [WSL](https://docs.microsoft.com/en-us/windows/wsl/).
 
+## Getting started
+For an overview of Palisade, start with root documentation [Palisade README](https://github.com/gchq/Palisade#readme).
+
 # Palisade Clients
 
 ## Client Implementations
+Implementation of a client will need to send requests into Palisade and process the response.  
+The workflow for this is expected to follow the pattern of first sending in a request for data.  
+This will return a response consisting of a token (a unique id) that can then be used a second request to retrieve the processed data.
+Client implementation will need to conform to this workflow, but the technology that is used and how the data is to be processed has by design meant to be left open for the implementations of the Palisade service.
```

Comment:
'left open for the implementations of the Palisade services.' As I was reading this I thought it was talking about the clients being written for the different processing technologies. Which meant that the Palisade services part threw me off here. The technology used for the clients would be dependent on what technology the user intends to use for follow on processing of the data.

Reply:
I don't even understand it.  I'll try again.